src/Microsoft.AspNetCore.Server.IntegrationTesting/breakingchanges.netcore.json

```diff
@@ -0,0 +1,78 @@
+[
+    {
+      "TypeId": "public abstract class Microsoft.AspNetCore.Server.Testing.ApplicationDeployer : Microsoft.AspNetCore.Server.Testing.IApplicationDeployer",
```

Comment:
The assembly name change happened before the baselines were generated. I don't think we need to update the baseline.